### DISCUSSION TITLE
Gordian System demo (airgapped signing, one click node, remote wallet over Tor)

### DISCUSSION THREAD

**Discussion by Fonta1n3** [BlockchainCommons/Gordian-Developer-Community#25] [Show and tell]
Hi all, I've been working with @ChristopherA @wolfmcnally  pretty hard getting all these pieces working together to form an easy to use stack for running a node, connecting remotely, easy to use multisig and easy to use airgapped signing. Would not be possible without Wolf McNally’s excellent work! This is a demo of the entire system (I am terrible at demos): 

https://m.youtube.com/watch?v=FCJ9QcNXjoU

We need testers and feedback! You can think of Gordian Server/Signer/Wallet as MVP's (minimum viable products), the UX and UI of all of these products needs a great deal of work but the functionality and ease of use is there. We would greatly appreciate you to engage and try these things out on testnet and raise issues on the respective repos with any and all feedback you may have.

**Comment by wolfmcnally** [comment]
I thought the demo was pretty good. Music mix was a bit high and threatened to bury the narration at times.
